### PR TITLE
Use score url from env for getScores

### PR DESCRIPTION
**Pull request by mktcode**
Using the existing `VITE_SCORES_URL` env var for calls of `snapshot.js`'s [`getScores` function](https://github.com/snapshot-labs/snapshot.js/blob/f2423f1221099f7c03b75853dad464cfe5c0faf1/src/utils.ts#L120) (for easier testing).

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]

**Comment by midgerate** [comment]
@mktcode is the purpose of the PR to remove the default score url from snapshot.js?

**Comment by mktcode** [comment]
@midgerate No, the purpose is just to make use of the new parameter, so that I can change it in my snapshot env file.

File: src/helpers/snapshot.ts [hunk 1]
```diff
@@ -65,7 +65,8 @@ export async function getResults(space, proposal, votes) {
         strategies,
         space.network,
         voters,
-        parseInt(proposal.snapshot)
+        parseInt(proposal.snapshot),
+        import.meta.env.VITE_SCORES_URL + '/api/scores',
```
Comment (bonustrack) [inline src/helpers/snapshot.ts:69]: I think the env var should be the full url, without /api/scores
```suggestion
        import.meta.env.VITE_SCORES_URL,
```

File: src/helpers/snapshot.ts [hunk 2]
```diff
@@ -105,7 +106,8 @@ export async function getPower(space, address, proposal) {
       strategies,
       space.network,
       [address],
-      parseInt(proposal.snapshot)
+      parseInt(proposal.snapshot),
+      import.meta.env.VITE_SCORES_URL + '/api/scores',
```
Comment (bonustrack) [inline src/helpers/snapshot.ts:110]:
```suggestion
      import.meta.env.VITE_SCORES_URL,
```

File: src/views/Delegate.vue [hunk 3]
```diff
@@ -142,7 +142,8 @@ async function getDelegatesWithScore() {
       delegationStrategy,
       space.value.network,
       delegatesAddresses,
-      'latest'
+      'latest',
+      import.meta.env.VITE_SCORES_URL + '/api/scores',
```
Comment (bonustrack) [inline src/views/Delegate.vue:146]:
```suggestion
      import.meta.env.VITE_SCORES_URL,
```

File: src/views/Playground.vue [hunk 4]
```diff
@@ -79,7 +79,8 @@ async function loadScores() {
       [strategyParams],
       form.value.network.toString(),
       form.value.addresses,
-      parseInt(form.value.snapshot)
+      parseInt(form.value.snapshot),
+      import.meta.env.VITE_SCORES_URL + '/api/scores',
```
Comment (bonustrack) [inline src/views/Playground.vue:83]:
```suggestion
      import.meta.env.VITE_SCORES_URL,
```

**Review by bonustrack — APPROVED** [review]

**Pull request merged**